news/_posts/translation/2020-05-12-do-you-speak-prestashop-may-2020.md

```diff
@@ -0,0 +1,131 @@
+It might be inconvenient for now but PrestaShop's developers team have in the backlog projects to make all new translations available for all shops from the 1.7.5 version and on, so let’s keep a positive mind! :sparkles:
+
+### :two_hearts: Special thanks to newcomers
+
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straightaway. For that reason, the PrestaShop team would like to thank the ones who rushed into the effort immediately! So lots of love to the dedicated: choefer, Çocuk Saatleri (cocuksaatleri), nenowasaleh, Pacho Gonzalez (Pachogzz), Puunaama1 and Stella Georgakopoulou (Stella_Geo). See you soon on Crowdin… and in PrestaShop :stuck_out_tongue_winking_eye:
+
+
+## A few stats
+
+* **37 members** were active on the project this month.
+* A total of **53,384 words have been translated** and **46,675 validated**.
+* All this in **28 different languages**.
+
+Thank you for your involvement!
+
+
+## Top contributors
+
+A lot of you are working every day on Crowdin to have PrestaShop available in many languages, and we can't thank you enough for your dedication! Here are the **most active translators and proofreaders** for May 2020.
+ 
+#### Top 10 translators in May:
+ 
+| |Translator | Language | # Words
+|-|---------- | -------- | ----------------
+ 1. | Rijad Osmanovic (rijado) | Bosnian | 29,098
+ 2. | mirmal | Esperanto | 7,892
+ 3. | cubache | Catalan | 4,058
+ 4. | Çocuk Saatleri (cocuksaatleri) | Turkish | 3,676
+ 5. | Rytis (Mangustux) | Lithuanian | 1,184
+ 6. | Rauno Riikman (weaver) | Finnish | 852
+ 7. | AmirRzd | Persian | 642
+ 8. | Zoran Tejic (zotamal) | Serbian (Latin) | 588
+ 9. | nenowasaleh | Swahili | 568
+10. | Stella Georgakopoulou (Stella_Geo) | Greek | 500
+ 
+ 
+#### Top 10 proofreaders in May:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+1. | Rijad Osmanovic (rijado) | Bosnian | 40,220
+ 2. | AmirRzd | Persian | 954
+ 3. | SeongHyeon Cho (jaymz9634) | Korean | 832
+ 4. | Gerardas (gerardas) | Lithuanian| 635
+ 5. | Zoran Tejic (zotamal) | Serbian (Latin) | 588
+ 6. | David Gonzalez (davidglezz) | Spanish | 537
+ 7. | Stamatis (breezer) | Greek | 512
+ 8. | Patrik Lindblom (extendus) | Swedish | 489
+ 9. | webdvl (megashopba) | Slovak; Czech | 455
+10. | Gabriel Tenita (ggedamed) | Romanian | 438
+ 
+Congrats, and welcome to our new top contributors :clap:
+ 
+Remember, you can see who's been contributing to our translation project thanks to the [Translators page](http://translators.prestashop.com/).
+ 
+ 
+## Complete translations
+ 
+#### Fully translated languages
+ 
+At the end of May 2020, PrestaShop 1.7.7 was fully available (= 100% translated and validated) in **11 languages**:
+ 
+| Danish | **Spanish** | **Italian** | **Dutch** | Portuguese, Brazil | Slovene | **Sweedish** | Chinese Traditional | **Greek** | **Slovak** | **French**|
```

Comment:
Watch out, there's a typo in the _Swedish_ word. Also, it's better to add them in alphabetical order.